stock_picking_backorder_to_sale/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===============================
+Stock Picking Backorder To Sale
+===============================
+
+Create a new sale order from picking and cancel this picking
+
+Usage
+=====
+
+To use this module:
+
+#. Go to *Warehouse > All Operations*
+#. Select a operation don't done with partner
```

Comment:
I'd suggest the second usage point to be expressed better something like "Select an undone operation with partner set"